### ISSUE TITLE
'az aro create' returns 'Request is not allowed in provisioningState 'Creating'.'

### ISSUE THREAD

**Issue by jyama4ta** [Azure/azure-cli#19114]
## Describe the bug
We get 'Request is not allowed in provisioningState 'Creating'.' error when try to create ARO by cloud shell.
It may be not waiting to finish creation.

**Command Name**
`az aro create`

**Errors:**
```
Request is not allowed in provisioningState 'Creating'.

```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az aro create --resource-group {} --name {} --vnet {} --master-subnet {} --worker-subnet {} --pull-secret {}`

## Environment Summary
```
Linux-5.4.0-1055-azure-x86_64-with-debian-10.2 (Cloud Shell)
Python 3.6.10
Installer: DEB

azure-cli 2.26.0 *

Extensions:
azure-firewall 0.11.0
ai-examples 0.2.5
ssh 0.1.5

```
## Additional Context
We could create an ARO cluster two days ago with same commands.

**Comment by yonzhan** [comment]
route to service team

**Comment by mjudeikis** [comment]
This is expected. If cluster create is in-progress we dont allow other write operations. 
Are you 100% sure where is not other create running?
```
Put any pre-requisite steps here...
az aro show -g {} -n {} # show existing ones 
az aro create --resource-group {} --name {} --vnet {} --master-subnet {} --worker-subnet {} --pull-secret {}
```

**Comment by jyama4ta** [comment]
Hi @mjudeikis , I'm sorry for my late response.
Expected behavior is on 'Running' state until finished creation. However, the commands returns 'Request is not allowed in provisioningState 'Creating'.' for that time.

However, I have checked it again on Cloud Shell and it is fixed on 2.27.0. So, I close this issue. Thank you for your time.
